server/views/components/transporter/index.config.yml

```diff
@@ -0,0 +1,108 @@
+context:
+  title: Latest articles
+  promos:
+  - modifiers: ['transporter-child']
+    headingLevel: h3
+    url: http://google.com
+    image:
+      sources:
+      - src: https://placehold.it/1200x900
+        size: '1200'
+      - src: https://placehold.it/900x600
+        size: '900'
+      - src: https://placehold.it/450x300
+        size: '450'
+    meta: 
+      type: article
+      date: 2 days ago
+    title: Oops!...I wrote a Britney blog post
+  - modifiers: ['transporter-child']
+    headingLevel: h3
+    url: http://google.com
+    image:
+      sources:
+      - src: https://placehold.it/1200x900
+        size: '1200'
+      - src: https://placehold.it/900x600
+        size: '900'
+      - src: https://placehold.it/450x300
+        size: '450'
+    meta: 
+      type: article
+      date: 2 days ago
+    title: Oops!...I wrote a Britney blog post
+  - modifiers: ['transporter-child']
+    headingLevel: h3
+    url: http://google.com
+    image:
+      sources:
+      - src: https://placehold.it/1200x900
+        size: '1200'
+      - src: https://placehold.it/900x600
+        size: '900'
+      - src: https://placehold.it/450x300
+        size: '450'
+    meta: 
+      type: article
+      date: 2 days ago
+    title: Oops!...I wrote a Britney blog post
+variants:
+  - name: Theme transporter
+    context:
+      type: theme
+      title: Mind control
+      introCopy: Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+      themeUrl: http://google.co.uk
+      promos:
+      - modifiers: ['transporter-child']
+        headingLevel: h3
+        url: http://google.com
+        image:
+          sources:
+          - src: https://placehold.it/1200x900
+            size: '1200'
+          - src: https://placehold.it/900x600
+            size: '900'
+          - src: https://placehold.it/450x300
+            size: '450'
+        meta: 
+          type: article
+          date:
+        title: life Before Death
+      - modifiers: ['transporter-child']
+        headingLevel: h3
+        url: http://google.com
+        image:
+          sources:
+          - src: https://placehold.it/1200x900
+            size: '1200'
+          - src: https://placehold.it/900x600
+            size: '900'
+          - src: https://placehold.it/450x300
+            size: '450'
+        meta: 
+          type: video
+          length: "1:47"
+          date:
+        title: Depressed in Dhariva
+      - modifiers: ['transporter-child']
+        headingLevel: h3
+        url: http://google.com
+        image:
+          sources:
+          - src: https://placehold.it/1200x900
+            size: '1200'
+          - src: https://placehold.it/900x600
+            size: '900'
+          - src: https://placehold.it/450x300
+            size: '450'
+        meta: 
+          type: article
+          date:
+        title: "Inspired: Housewives & alchemists around a long table"
+
```

Comment:
Do you think it might be useful to start creating JS models for these so they become applicable everywhere?
e.g. https://github.com/wellcometrust/wellcomecollection.org/blob/master/server/views/components/image/index.config.js

@davidpmccormick - might be worth thinking about for the work you're doing.